src/runtime/server.ts

```diff
@@ -159,6 +159,14 @@ class SocketRuntimeClient extends RuntimeClient {
       dbs["editor"] = new BrowserEditorDatabase();
       dbs["inspector"] = new BrowserInspectorDatabase();
     }
+    // in the case where we're running on both the client and the server,
+    // we need to add a browser-session bag and also make our local browser bag
+    // be a normal database that way we don't send UI that is already being handled
+    // by the client
+    if(config.runtimeOwner === Owner.both) {
```

Comment:
We could potentially make client-server equipped code run on either/or as well if we always add this and just let it play telephone with itself. I don't know that doing so is particularly useful, but it may be good for testing at least.